Library/Formula/pdfgrep.rb

```diff
@@ -15,8 +15,16 @@ class Pdfgrep < Formula
   depends_on "pkg-config" => :build
   depends_on "poppler"
 
+  if build.head?
+    depends_on :automake
+    depends_on :autoconf
+    depends_on "asciidoc" => :build
+  end
```

Comment:
Seems like you were just telling me about those symbols a little while ago... :astonished: